extensions/Cake.AutoRest.yml

```diff
@@ -9,4 +9,5 @@ Description: Cake addin for generating client code from compatible API definitio
 Categories:
 - azure
 - swagger
-TargetCakeVersion: 0.18.0
+TargetCakeVersion: 0.13.0
```

Comment:
Why the change here?

Reply:
Because we changed the logic to ensure we analyze the most recent "stable" release:

![image](https://user-images.githubusercontent.com/112710/101052895-2689f600-3555-11eb-9bc0-9d9c2b64e0ec.png)